templates/template/Makefile

```diff
@@ -1,8 +1,13 @@
-CHARTS_BUILD_SCRIPT_VERSION := fdf0565
-
 pull-scripts:
 	./scripts/pull-scripts
 
+{{ if (eq .Template "staging") -}}
+
+release:
+	./scripts/release-assets
+
+{{ end -}}
```

Comment:
Are you go-templating a makefile ? Guessing this gets pushed out by the go code and then executed inside container.

Reply:
Yes, everything within `templates/template` is a Go template based on the `configuration.yaml`. That's how we ship different files to different branches